source/howto_import-mail.rst

```diff
@@ -0,0 +1,174 @@
+you to use for any mail client:
+
+.. code-block:: ini
+
+    --host1 imap.example.com \
+    --user1 info@example.com \
+    --ssl1 \
+
+If your old mailserver is located at Gmail, you should use the following *instead* (this will provide the
+correct settings for Google Mails):
+
+.. code-block:: ini
+
+    --gmail1 \
+    --user1 info@example.com \
+
+Then you need to define the target mailbox on your Uberspace. Keep in mind to replace ``info``
+with the correct mailbox.
+
+.. code-block:: ini
+
+    --host2 localhost \
+    --user2 info@$USER.uber.space \
+    --ssl2 \
+
+Copy all configured lines at once to the command line like:
+
+.. code-block::
+
+    [isabell@stardust ~]$ imapsync \
+    --logdir ~/logs/imapsync \
+    --host1 imap.example.com \
+    --user1 info@example.com \
+    --ssl1 \
+    --host2 localhost \
+    --user2 info@$USER.uber.space \
+    --ssl2 \
+
+After executing the command, you will be asked for the old and new mail address passwords to enter,
+then ``imapsync`` should take some time and import the mail and folder structure from your old mail account.
+
+.. tip::
+    The command tool ``imapsync`` is very powerful with a lot of configuration options. While the settings explained here
+    should suit for the most cases, you might want or need to change some of them. Have a look at the `repo <https://github.com/imapsync/imapsync>`_.
+
+
+Set up mail domain
+------------------
+
+After importing the mails you can continue to switch the domain to the new location. First add the domain to your Uberspace
+so it will be recognised by the host:
+
+.. code-block::
+
+    [isabell@stardust ~]$ uberspace mail domain add example.com
+    The mailserver's configuration has been adapted.
+    Now you can use the following record for your DNS:
+        MX  -> stardust.uberspace.de.
+        TXT -> v=spf1 include:spf.uberspace.de ~all
+
+    The trailing dot may be skipped, if the interface does not accept it.
+
+    [isabell@stardust ~]$
+
+This will provide you the neccessary ``DNS`` records you have to set up at your **domain registrar**. You will need to replace
+the ``MX`` record of your old mailserver to fully connect your domain to your Uberspace.
+
+After changing the ``DNS`` records, it might take a short time until our server could check for the correct settings,
+run the Uberspace command to get the current status:
+
+.. code-block::
+
+    [isabell@stardust ~]$ uberspace mail domain list
+    example.com DNS INVALID (checked 2022-02-08 20:36)
+    isabell.uber.space
+
+    [isabell@stardust ~]$
+
+When the ``DNS INVALID`` warning is not shown any longer, the settings are correct and mails might already be delivered to your new Uberspace.
+
+.. warning::
+    Mails sent to your domain within 2 hours after updating the DNS records could still be sent to your old mail provider, because other mailservers
+    will take longer to update their DNS resolvers. You should check your old mailbox on this.
+
+
```

Comment:
> Other providers use different named spam folders, people may need to move mails around. 

move their spamfolders around? mmh I dont know if this is maybe too much micromanagement which would complicate the howto. maybe we could put it to the troubleshooting section?

> Also a short paragraph about IMAP access or a hint to the webmailer would be nice maybe?

maybe this could take place within a new "after migrating" section which sums up some articles in the manual etc, I will have a look

> Migration usually is not done by just moving mails around. People use calendars and contacts with other providers. I think a hint for a nextcloud or something very easy to setup would be nice too.

this could also be part for this section, but I would say we should handle this in additional migration howtos

Reply:
I put all of this to the troubleshooting section